.ci/cico_build_and_deploy_custom_rh-che.sh

```diff
@@ -5,38 +5,33 @@
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 
+# ERROR CODES:
+# 1 - missing file
+# 2 - missing credentials
+# 3 - OpenShift login failed
+# 4 - command execution failed
+
```

Comment:
This file is only ment to be used by prcheck job right? Why don't you rename the script so that it is straightforward (something like `cico_rhche_prcheck.sh`)?